ethereum/chain.go

```diff
@@ -81,12 +80,12 @@ func NewEth(
 	ctx context.Context, log *slog.Logger, m *metrics.PrometheusMetrics,
 	websocketurl, rpcurl string, testnet bool, overrides Overrides,
 ) (*Eth, error) {
-	webSocketClient, err := dialClient(ctx, log, websocketurl)
+	webSocketClient, err := dialClient(ctx, log, websocketurl, "websocket")
 	if err != nil {
 		return nil, fmt.Errorf("failed to dial websocket client: %v", err)
 	}
 
-	rpcClient, err := dialClient(ctx, log, rpcurl)
+	rpcClient, err := dialClient(ctx, log, rpcurl, "RPC")
```

Comment:
just a suggestion but since there are only two valid values that are passed in for the `clientType` parameter these could be constants or enums which would remove the magic strings and make the code more readable/maintainable.

Reply:
Good idea! [`b51a539` (#15)](https://github.com/noble-assets/jester/pull/15/commits/b51a53919def955f5fbb6d40153a7f4668b52e37)